cfme/exceptions.py

```diff
@@ -277,6 +277,10 @@ class NeedleNotFoundInLog(CFMEException):
     """Raised when log doesnt't contain needle"""
 
 
+class RestLookupError(CFMEException):
+    """raised when lookup of a rest entity fails"""
+
+
```

Comment:
Do we not want to define any specific error message here?

Reply:
Not really, whatever exception message occurred in the underlying exception class would carry through.